docs/readmes/orc8r/upgrade_1_4.md

```diff
@@ -58,8 +58,8 @@ module orc8r-app {
   source = "github.com/magma/magma//orc8r/cloud/deploy/terraform/orc8r-helm-aws?ref=v1.4"
   # ...
   orc8r_chart_version   = "1.5.11"
-  orc8r_tag             = "MAGMA_TAG"       # from build step, e.g. v1.4.0
-  orc8r_deployment_type = DEPLOYMENT_TYPE   # valid options are: ["fwa", "federated_fwa", "all"]
+  orc8r_tag             = "MAGMA_TAG"  # from build step, e.g. v1.4.0
+  orc8r_deployment_type = "all"        # valid options: ["fwa", "federated_fwa", "all"]
```

Comment:
can we make this `fwa` instead? As that is what we anticipate folks would do?

Reply:
The other change I was requesting was: 

If this orc8r is a 2nd instance on this AWS **account**, then the service linked role `AWSServiceRoleForAmazonElasticsearchService` may already exist and you may see an error when running the final `terraform apply`. In such events, the variable `deploy_elasticsearch_service_linked_role` should be set to false under module `orc8r`.

Reply:
Ah I see. I think since, for now at least, we explicitly mention we only support a single deployment per AWS account (hacky but wygd), perhaps this information belongs in the FAQ page. Thoughts?

In any case I'll leave it as a separate PR. Are you down to take that on since you have more context @sudhikan ?